pkg/daemon/update_test.go

```diff
@@ -64,104 +66,184 @@ func TestUpdateOS(t *testing.T) {
 // TestReconcilable attempts to verify the conditions in which configs would and would not be
 // reconcilable. Welcome to the longest unittest you've ever read.
 func TestReconcilable(t *testing.T) {
+	oldIgnCfg := ctrlcommon.NewIgnConfig()
 	// oldConfig is the current config of the fake system
 	oldConfig := &mcfgv1.MachineConfig{
 		Spec: mcfgv1.MachineConfigSpec{
-			Config: ctrlcommon.NewIgnConfig(),
+			Config: runtime.RawExtension{
+				Raw: helpers.MarshalOrDie(oldIgnCfg),
+			},
 		},
 	}
+	newIgnCfg := ctrlcommon.NewIgnConfig()
+
+	// Set improper version
+	newIgnCfg.Ignition.Version = "3.0.0"
```

Comment:
("999.0.0" or something clearly too high?)

Reply:
3.0.0 is a version that actually exists and that is clearly not yet supported by MCO

Reply:
In other words, as soon as MCO will add support for it, this test will break. Or is that the purpose of this test?

Reply:
This test will indeed have to change once support is added.
However, right now it is important to check v3 is not accepted.

Reply:
agree with @LorbusChris this should stay  as-is